index.html

```diff
@@ -7,29 +7,46 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title> Portfolio </title>
   <link rel="stylesheet" href="./style.css">
+  
 </head>
-
 <body>
   <main>
     <header>
       <nav>
         <div class="desktop-menu">
           <ul>
             <li>
-              <a href="">Portfolio</a>
+              <a href="#">Portfolio</a>
             </li>
             <li>
-              <a href="">About</a>
+              <a href="#">About</a>
             </li>
             <li>
-              <a href="">Contact</a>
+              <a href="#">Contact</a>
             </li>
           </ul>
         </div>
-        <div class="mobile-menu-icon">
-          <img src="./images/Union.svg" alt="Bar Image">
+      </nav>
+      <nav class="navbar">
+        
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Contact</a>
+          </li>
+        </ul>
+        <div class="hamburgar">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
         </div>
       </nav>
```

Comment:
- [ ] Kindly make sure when the mobile menu opened, it will be fixed even if I scroll, take a look at the following image when I scroll the other sections appear
>![image](https://github.com/newtonah0/Mobile-Portfolio/assets/53402600/416dd36c-0388-48f2-b630-792f822a98f5)

- [ ] Also consider when clicking on any nav link should navigate to a specific part on the screen